### ISSUE TITLE
Ability to customize the status bar

### ISSUE THREAD

**Issue by gijsyo** [Novik/ruTorrent#2771]
### Is your feature request related to a problem?

Would there be a way to make the status bar customizable, let's say like the columns, through a context menu? It's too wide for my laptop screen and I have no need for the CPU monitor and the open socket info so I would like to remove them. The CPU monitor was easy to remove by deleting the plugin dir but it seems more sane to be able to (de)select components for the status bar.

### Describe the solution you'd like

Anything would be an improvement. My preference is for a right-click menu like the column selector.

**Comment by gijsyo** [comment]
> Hello @gijsyo, the settings to remove items from the status bar are disorganized in ruTorrent. However, they are still present.
> 
>     1. You can go to `settings -> format -> uncheck open status in status bar` to remove the open socket info.
> 
>     2. You can disable the `cpuload` plugin in the plugins tab without removing it.
> 
> 
> Do you think we should add a `status bar` tab to the settings menu to customize it? I believe this will address your feedback. I would like to hear about @jevenski 's thoughts for setting a milestone of `v5.2` for this improvement.

Thanks for the tip. Of course it worked. To me it seems logical to have a status bar settings menu if it's not too much trouble.

Maybe get a plugins section in the settings as well instead of its own button?

**Comment by jevenski** [comment]
Yes, in the long term, context menu should be disabled completely on mobile, after we have implemented equivalent replacement to achieve all functionalities.

On desktop, it is still the best way to perform most actions, in my opinion. This does introduce an inconsistency between desktop and mobile, and thus pose a challenge in designing and maintenance.
